src/server/pkg/storage/fileset/option.go

```diff
@@ -52,6 +54,12 @@ func WithMaxOpenFileSets(max int) StorageOption {
 // UWriterOption configures an UnorderedWriter.
```

Comment:
I would prefer that we spell this out here, so `UWriterOption` -> `UnorderedWriterOption`. I think we will probably want to minimize the abbreviation of configuration.